ServicesDelete.php

```
@@ -0,0 +1,237 @@
+                if (!$this->promptAndValidatePassword()) {
+                    continue;
+                }
+                $toDelete->delete();
+                $this->info('Server deleted successfully.');
+            }
+        }
+    }
+
+    private function deleteApplication()
+    {
+        $applications = Application::all();
```

PHPMD (2.15.0)

Avoid using static access to class '\App\Models\Application' in method 'deleteApplication'. (undefined)

(StaticAccess)